pymc3/sampling.py

```diff
@@ -2006,6 +2008,14 @@ def init_nuts(
         cov = np.atleast_1d(pm.trace_cov(init_trace))
         start = list(np.random.choice(init_trace, chains))
         potential = quadpotential.QuadPotentialFull(cov)
+    elif init == "adapt_full":
+        start = [model.test_point] * chains
+        mean = np.mean([model.dict_to_array(vals) for vals in start], axis=0)
+        init_trace = pm.sample(
+            draws=n_init, step=pm.NUTS(), tune=n_init // 2, random_seed=random_seed
+        )
```

Comment:
These three lines are unnecessary.

I'm guessing that you copied this from the `init == "nuts"` case, where we actually run a Markov chain with NUTS, and use the sample covariance from that chain as our estimate of the mass matrix. This isn't what we're doing here: `"adapt_full"` has a lot more in common with `"adapt_diag"`, with the only difference being that we're now using a dense matrix instead of a diagonal matrix.

```suggestion
```

Reply:
hey @eigenfoo I have update the PR, please have a look